models/Experiment.model.js

```diff
@@ -1,122 +1,124 @@
-const {Schema, model, ObjectId,isValidObjectId} = require("mongoose");
+const { Schema, model, ObjectId, isValidObjectId } = require("mongoose");
 const iso = require("iso-3166-1"); // used to validate country code
 
 const experimentSchema = new Schema(
-    {
-        name: {type: String, required: true},
-        accountId: {type: ObjectId, required: true},
-        type: {
-            type: String,
-            enum: ["f-f", "a-b"],
-            required: true,
-        },
-        testAttributes: {
-            location: {
-                type: [String],
-                validate: {
-                    validator: countryValidator,
-                    message: () => `Invalid country code`,
-                },
-            },
-            device: {
-                type: [String],
-                validate: {
-                    validator: deviceValidator,
-                    message: () => `Invalid device`,
-                },
-                lowercase: true,
-                trim: true,
-            },
-            browser: [String],
+  {
+    name: { type: String, required: true },
+    accountId: { type: ObjectId, required: true },
+    type: {
+      type: String,
+      enum: ["f-f", "a-b"],
+      required: true,
+    },
+    testAttributes: {
+      location: {
+        type: [String],
+        validate: {
+          validator: countryValidator,
+          message: () => `Invalid country code`,
```

Comment:
did you make any changes to the Experiment.model??
for some reason the code dosent exactly line up next to each other so it is difficult to compare

Reply:
I wrote down the changes everyone should make in lines 1 and 94. It was after I've talk to you and peer yesterday, he made some minor mistakes with the schema. Also Shir made her refactor changes, i wrote about it yesterday on slack